src/target/stm32l4.c

```diff
@@ -149,6 +150,9 @@ const command_s stm32l4_cmd_list[] = {
 #define DUAL_BANK     0x80U
 #define RAM_COUNT_MSK 0x07U
 
+/* TODO: add block size constants for other MCUs */
+#define STM32U5_FLASH_BLOCK_SIZE	0x2000
```

Comment:
Please include a `U` suffix on this constant - this makes it unsigned and prevents silent bugs down the road.